web-registry/src/pages/CreditClassDetailsSimple.tsx

```diff
@@ -0,0 +1,313 @@
+
+interface CreditDetailsProps {
+  dbClass: any;
+  ledgerClass: ClassInfo;
+  metadata?: any;
+}
+
+const useStyles = makeStyles<Theme>((theme: Theme) => ({
+  sectionPadding: {
+    [theme.breakpoints.up('sm')]: {
+      paddingTop: theme.spacing(30),
+      paddingBottom: theme.spacing(30),
+    },
+    [theme.breakpoints.down('sm')]: {
+      paddingTop: theme.spacing(20),
+      paddingBottom: theme.spacing(20),
+    },
+  },
+  label: {
+    fontSize: theme.typography.pxToRem(12),
+    color: theme.palette.primary.contrastText,
+    letterSpacing: '1px',
+    lineHeight: '15px',
+    marginBottom: theme.spacing(2),
+  },
+  box: {
+    display: 'flex',
+    flexDirection: 'column',
+    borderRadius: 5,
+    border: `1px solid ${theme.palette.grey[100]}`,
+    background: theme.palette.primary.main,
+    padding: theme.spacing(11, 5),
+    overflow: 'auto',
+  },
+  link: {
+    color: theme.palette.secondary.main,
+    fontWeight: 700,
+  },
+  description: {
+    [theme.breakpoints.up('sm')]: {
+      fontSize: theme.typography.pxToRem(22),
+    },
+    [theme.breakpoints.down('sm')]: {
+      fontSize: theme.typography.pxToRem(18),
+    },
+  },
+  marginBottom: {
+    [theme.breakpoints.up('sm')]: {
+      marginBottom: theme.spacing(12),
+    },
+    [theme.breakpoints.down('sm')]: {
+      marginBottom: theme.spacing(8.75),
+    },
+  },
+  sidebarItemMargin: {
+    [theme.breakpoints.up('sm')]: {
+      marginBottom: theme.spacing(8.75),
+    },
+    [theme.breakpoints.down('sm')]: {
+      marginBottom: theme.spacing(8),
+    },
+    '&:last-child': {
+      marginBottom: 0,
+    },
+  },
+  arrow: {
+    fontSize: theme.typography.pxToRem(16),
+  },
+}));
+
+const CreditClassDetailsSimple: React.FC<CreditDetailsProps> = ({
+  dbClass,
+  ledgerClass,
+  metadata,
+}) => {
+  const styles = useStyles();
+
+  const getAllProjects = (): JSX.Element => {
+    const projects = dbClass?.projectsByCreditClassId?.nodes;
+
+    return projects && projects.length > 0 ? (
+      <div className="topo-background-alternate">
+        <MoreProjectsSection
+          classes={{ root: styles.sectionPadding, title: styles.title }}
+          title="Projects"
+          projects={projects}
+        />
+      </div>
+    ) : (
+      <></>
+    );
+  };
+
+  const ApprovedMethodologies: React.FC<{ methodologies: any[] }> = ({
+    methodologies,
+  }): JSX.Element => {
+    const count = methodologies?.length;
+    const firstMethodology = methodologies?.[0];
+
+    return count > 0 ? (
+      <LineItem
+        label="approved methodologies"
+        data={
+          <Box>
+            <Description
+              className={styles.description}
+              key={firstMethodology?.['schema:name']}
+            >
+              {firstMethodology?.['schema:name']}
+            </Description>
+            {count > 1 && (
+              <Link
+                sx={{ display: 'flex', alignItems: 'center' }}
+                href="TODO"
```

Comment:
This URL is not defined in the schema standard yet. I have a pending question about how it will show up here, but easy to fix once it is decided

Reply:
updated to WIP standard, but this may still need to be updated.